Forgive line-ending mismatches in the python test suite.

The new python test suite is failing on MSYS2 when cross compiled on Fedora and running on Windows.  This is caused by line-ending mismatches.

This is discussed here: https://github.com/Exiv2/exiv2/issues/922#issuecomment-610275806

I believe it's possible to modify the sample programs to output LF by calling `setmode()` and putting stdout into binary mode.  https://stackoverflow.com/questions/5813301/how-c-output-lf-to-stdout-without-being-changed-to-cr-lf

Modifying the code in all our sample programs is a brutal approach when the alternative is to modify the python test suite.  The bash test suite is executing correctly as it performs executes `diff` with `diffargs="--strip-trailing-cr"`

I used unix2dos to modify all the python files and was surprised when that fix was not successful.

My fix is good and reveals a new issue.  Many tests fail as follows:
```
======================================================================
FAIL: test_run (bugfixes.redmine.test_issue_841.ThrowOnInput)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rmills/temp/0.27-maintenance/tests/system_tests.py", line 632, in test_run
    self.compare_stderr(i, command, processed_stderr, stderr)
  File "/home/rmills/temp/0.27-maintenance/tests/system_tests.py", line 755, in compare_stderr
    msg="Standard error does not match"
  File "/home/rmills/temp/0.27-maintenance/tests/system_tests.py", line 726, in _compare_output
    expected, got, msg=msg
AssertionError: 'Exiv[32 chars]file C:/msys64/home/rmills/temp/0.27-maintenan[55 chars]ta\n' != 'Exiv[32 chars]file /home/rmills/temp/0.27-maintenance/test/d[46 chars]ta\n'
- Exiv2 exception in print action for file C:/msys64/home/rmills/temp/0.27-maintenance/test/data/exiv2-bug841.png:
?                                          ---------
+ Exiv2 exception in print action for file /home/rmills/temp/0.27-maintenance/test/data/exiv2-bug841.png:
  Failed to read image data
 : Standard error does not match
```
There's a path issue here.  You can run a single test with the following syntax:
```
$ cd <exiv2dir>/tests
$ python3 runner.py --verbose bugfixes/redmine/test_issue_841.py 
```

The python test suite is excellent.  Great Job: @D4N  I'll continue to investigate.

--- a/tests/system_tests.py
+++ b/tests/system_tests.py
@@ -12,7 +12,7 @@ import string
 import unittest
 
 
-if sys.platform in [ 'win32', 'msys', 'cygwin' ]':
+if sys.platform in [ 'win32', 'msys', 'cygwin' ]:
     #: invoke subprocess.Popen with shell=True on Windows
     _SUBPROCESS_SHELL = True
 

--- a/tests/system_tests.py
+++ b/tests/system_tests.py
@@ -12,7 +12,7 @@ import string
 import unittest
 
 
-if sys.platform == 'win32':
+if sys.platform in [ 'win32', 'msys', 'cygwin' ]':
     #: invoke subprocess.Popen with shell=True on Windows
     _SUBPROCESS_SHELL = True
 

